### DISCUSSION TITLE
What 🌊 separates 🇹🇷 from 🇬🇷?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#899] [Galaxy Trivia] [answered]
**Bonus:** How deep is it?

**Comment by peterbaccaro** [comment, marked as answer]
The sea that separates Turkey from Greece is called the **Aegean Sea**. 

The average depth of the Aegean Sea is around 220 meters (720 feet). However, it's important to note that the depth can vary significantly in different areas. The **maximum depth in the Aegean Sea reaches approximately 3,543 meters (11,624 feet)** in the central part of the sea, known as the Aegean Abyssal Plain.
